src/mainboard/dell/snb_ivb_workstations/default.fmd

```diff
@@ -0,0 +1,24 @@
+FLASH@0xff400000 12M {
+	SI_ALL@0x0 0x600000 {
+		SI_DESC@0x0 0x1000
+		SI_GBE@0x1000 0x4000
+		SI_ME@0x5000 0x5fb000
+	}
+	UNUSED@0x600000 0x200000
+	SI_BIOS@0x800000 0x400000 {
+		MISC_RW@0x0 0x84000 {
+			SMMSTORE(PRESERVE)@0x0 0x40000
+			RW_MRC_CACHE@0x40000 0x10000
+			CONSOLE@0x50000 0x20000
+			RW_VPD(PRESERVE)@0x70000 0x4000
+			RW_ELOG(PRESERVE)@0x74000 0x10000
+		}
+		WP_RO@0x84000 0x37c000 {
+			RO_VPD(PRESERVE)@0x0 0x4000
+			RO_SECTION@0x4000 0x378000 {
+				FMAP@0x0 0x800
+				COREBOOT(CBFS)@0x800 0x377800
+			}
+		}
+	}
+}
```

Comment:
```suggestion
FLASH@0xff400000 12M {
	SI_ALL 6M {
		SI_DESC 4K
		SI_GBE 16K
		SI_ME
	}
	UNUSED 2M
	SI_BIOS 4M {
		MISC_RW 384K {
			SMMSTORE(PRESERVE) 256K
			RW_MRC_CACHE 64K
			RW_ELOG(PRESERVE) 64K
		}
		WP_RO  {
			FMAP 2K
			COREBOOT(CBFS)
		}
	}
}
```
A bit simpler and more readable layout

Reply:
Fixed in [this commit](https://github.com/Dasharo/coreboot/pull/541/commits/280197799f4acde50801f94734ed5b179c715ccf)